### DISCUSSION TITLE
change blockchain folder path to another drive (like D:)

### DISCUSSION THREAD

**Discussion by Meloman-zz** [DeFiCh/app#819] [Q&A] [answered]
Hi,

On my computer, I have a 128GB SSD for C: and 1TB HDD for D: !

When I choose to install my app 2.3.3 on "D:\DeFi Wallet", the sync will still download the blockchain to c:\users\johndoe\AppData\Roaming. And I won't. How can we change path for DeFi Blockchain folder?

Thanks

**Comment by nico89-exp** [comment, marked as answer]
Hi @Meloman-zz this helped other users to do so.

https://defichain-wiki.com/wiki/Blockchain_folder

**Reply by Meloman-zz** [reply]
yeah thanks, exactly what I need.

I hope this will be a feature in the app preferences in a future.

**Reply by izzycsy** [reply]
In the future, we'll be reducing the storage required. The team is working on it. Thank you for your understanding. 

Thanks @nico89-exp !
